csvcubed/csvcubed/models/inspectsparqlresults.py

```diff
@@ -0,0 +1,209 @@
+"""
+Inspect SPARQL query results
+----------------------------
+"""
+
+from os import linesep
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Dict, List
+from csvcubed.utils.qb.components import get_printable_component_property
+from rdflib import URIRef
+
+from rdflib.query import ResultRow
+
+from csvcubed.utils.sparql import none_or_map
+from csvcubed.utils.printable import (
+    get_printable_list_str,
+    get_printable_tabular_list_str,
+    get_printable_tabular_str,
+)
+
+
+@dataclass()
+class CatalogMetadataSparqlResult:
+    """
+    Model to represent result of the `select_csvw_catalog_metadata` sparql query.
+    """
+
+    sparql_result: ResultRow
+
+    @property
+    def output_str(self) -> str:
+        formatted_landing_pages: str = get_printable_list_str(self.landing_pages)
+        formatted_themes: str = get_printable_list_str(self.themes)
+        formatted_keywords: str = get_printable_list_str(self.keywords)
+        formatted_description: str = self.description.replace(linesep, f"{linesep}\t\t")
+        return f"{linesep}\t- Title: {self.title}{linesep}\t- Label: {self.label}{linesep}\t- Issued: {self.issued}{linesep}\t- Modified: {self.modified}{linesep}\t- License: {self.license}{linesep}\t- Creator: {self.creator}{linesep}\t- Publisher: {self.publisher}{linesep}\t- Landing Pages: {formatted_landing_pages}{linesep}\t- Themes: {formatted_themes}{linesep}\t- Keywords: {formatted_keywords}{linesep}\t- Contact Point: {self.contact_point}{linesep}\t- Identifier: {self.identifier}{linesep}\t- Comment: {self.comment}{linesep}\t- Description: {formatted_description}"
+
+    def __post_init__(self):
+        result_dict = self.sparql_result.asdict()
+
+        self.title: str = str(result_dict["title"])
```

Comment:
I'm not sure that these get counted as dataclass fields since they're not defined in the standard location.